### PR TITLE
Fixing all issues under Django 2.0

### PR DESCRIPTION
**Pull request by kakulukia**
this will probably break some older Django versions, but support for those might also be dropped since they are kinda ......... old :)

### CODE REVIEW

**Comment by kakulukia** [comment]
Also fixing my own issue https://github.com/alsoicode/django-maintenancemode-2/issues/8

**Comment by kakulukia** [comment]
simplifying editing the readme :)

**Comment by alsoicode** [comment]
Thank you. My apologies for the delay on this; it's been a busy week already.

File: maintenancemode/models.py [hunk 1]
```diff
@@ -24,12 +24,12 @@ def ignored_url_patterns(self):
         qs = self.ignoredurl_set.values_list(
             "pattern", flat=True
             )
-        return map(str, qs)
+        return list(qs)
```
Comment (alsoicode) [inline maintenancemode/models.py:27]: This is already a list when returned from `.values_list()`. Was there a particular reason for passing `qs` to `list()` ?

File: README.rst [hunk 2]
```diff
@@ -34,7 +34,7 @@ Supported Python Versions
 Supported Django Versions
 -------------------------
 
--  1.7, 1.8, 1.9, 1.10, 1.11
+-  1.8, 1.9, 1.10, 1.11, 2.0
```
Comment (alsoicode) [inline README.rst:37]: I'll probably make some adjustments to the readme to have people use this version for Django 2 or higher and use the previous version for < 2.

File: maintenancemode/models.py (hunk 1 above)
Comment (kakulukia) [reply maintenancemode/models.py:27]: Manager methods always return querysets unless explicitly converted somewhere like here.

File: README.rst (hunk 2 above)
Comment (kakulukia) [reply README.rst:37]: As you wish. 🤩

File: maintenancemode/models.py (hunk 1 above)
Comment (alsoicode) [reply maintenancemode/models.py:27]: Doh. That's correct. I don't work with Django very much anymore, and completely forgot.

**Comment by alsoicode** [comment]
I will do my best to get a release out today or tomorrow.

**Comment by kakulukia** [comment]
Please wait with that relese, because i need that mentions staff exclusion option. Will add that today.

**Comment by alsoicode** [comment]
Sure, is no problem at all.

**Pull request merged**